src/Plugin/Field/FieldFormatter/FileSizeFormatter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\file_entity\Plugin\Field\FieldFormatter\FileSizeFormatter.
+ */
+
+namespace Drupal\file_entity\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Render\Element;
+
+/**
+ * Implementation of the 'filesize' formatter for the file_entity files.
+ *
+ * @FieldFormatter(
+ *   id = "file_size",
+ *   label = @Translation("File Size"),
+ *   field_types = {
+ *     "integer"
+ *   }
+ * )
+ */
+class FileSizeFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function isApplicable(FieldDefinitionInterface $field_definition) {
+    // This formatter is only available for files.
+    return $field_definition->getTargetEntityTypeId() == 'file';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items) {
+    $elements = array();
+    $files = [$items->getEntity()];;
```

Comment:
array() on one line, [] on the next, please be consistent (IMO [] is nicer than array())